### PR TITLE
Feat: [OV-39585] Initial release of carousel

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Carousel/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import Button from '@igloo-ui/button';
+import Carousel from '@igloo-ui/carousel';
+
+const SLIDE_NUM = 3;
+const [selected, setSelected] = React.useState(0);
+const [secondaryActionText, setSecondaryActionText] = React.useState('Cancel');
+const [primaryActionText, setPrimaryActionText] = React.useState('Next');
+
+const handlePageChange = (index) => {
+  if (index < SLIDE_NUM - 1) {
```
Comment (franckgaudin) [inline packages/Carousel/README.md:37]: This part of the doc. can be simplified by removing the `useState()` and placing the logic in the button.

Example: 
```
const handlePageChange = (index: number) => {
  setSelected(index);
};
```

File: packages/Carousel/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Then to use the component in your code just import it!
+
+```jsx
+import Button from '@igloo-ui/button';
+import Carousel from '@igloo-ui/carousel';
+
+const SLIDE_NUM = 3;
+const [selected, setSelected] = React.useState(0);
+const [secondaryActionText, setSecondaryActionText] = React.useState('Cancel');
```
Comment (franckgaudin) [inline packages/Carousel/README.md:33]:
```suggestion

```

File: packages/Carousel/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+const [primaryActionText, setPrimaryActionText] = React.useState('Next');
```
Comment (franckgaudin) [inline packages/Carousel/README.md:34]:
```suggestion

```

File: packages/Carousel/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    handlePageChange(selected - 1);
+  }
+};
+
+<Carousel
+  onPageChange={handlePageChange}
+  currentSlide={selected}
+  primaryAction={
+    <Button appearance={'primary'} onClick={handlePrimaryActionClick}>
+      {primaryActionText}
```
Comment (franckgaudin) [inline packages/Carousel/README.md:67]:
```suggestion
{selected < SLIDE_NUM - 1 ? 'Next' : 'Done'}
```

File: packages/Carousel/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    </Button>
+  }
+  secondaryAction={
+    <Button appearance={'ghost'} onClick={handleSecondaryActionClick}>
+      {secondaryActionText}
```
Comment (franckgaudin) [inline packages/Carousel/README.md:72]:
```suggestion
      {selected > 0 ? 'Prev' : 'Cancel'}
```

**Review by franckgaudin — DISMISSED** [review]

File: packages/Carousel/src/carousel.scss [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+:root {
+  /* Default */
+  --ids-carousel-font-family: #{tokens.$primary-font-family};
+  --ids-carousel-font-size: #{tokens.$font-size-4};
+  --ids-carousel-nav-margin: #{tokens.$space-2} -#{tokens.$space-2} -#{tokens.$space-2};
+
+  /* Bullets */
+  --ids-carousel-bullet-color: #{tokens.$grey-300};
+  --ids-carousel-bullet-focus: #{tokens.$focus};
```
Comment (ofrogon) [inline packages/Carousel/src/carousel.scss:12]: `--ids-carousel-bullet-focus` is unused

Comment (vicky-comeau) [reply packages/Carousel/src/carousel.scss:12]: Thanks, I put it where it belongs on line 91

**Review by ofrogon — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**